vm/src/stdlib/dis.rs

```diff
@@ -17,11 +18,25 @@ fn dis_disassemble(co: PyObjectRef, vm: &VirtualMachine) -> PyResult {
     Ok(vm.get_none())
 }
 
+fn dis_compiler_flag_names(vm: &VirtualMachine) -> PyObjectRef {
+    let dict = vm.ctx.new_dict();
+    for (name, flag) in CodeFlags::NAME_MAPPING {
+        dict.set_item(
+            &vm.ctx.new_int(flag.bits()),
+            vm.ctx.new_str(name.to_string()),
+            vm,
+        )
+        .unwrap();
+    }
+    dict.into_object()
+}
+
 pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
     let ctx = &vm.ctx;
 
     py_module!(vm, "dis", {
         "dis" => ctx.new_rustfunc(dis_dis),
-        "disassemble" => ctx.new_rustfunc(dis_disassemble)
+        "disassemble" => ctx.new_rustfunc(dis_disassemble),
+        "COMPILER_FLAG_NAMES" => dis_compiler_flag_names(vm),
```

Comment:
Can you add test for this in dismod.py?

Reply:
What kind of test? It's essentially just a dict mapping ints to strings.